packages/@aws-cdk/aws-eks/lib/cluster.ts

```diff
@@ -199,7 +200,7 @@ export interface ICluster extends IResource, ec2.IConnectable {
    * @param chart the cdk8s chart.
    * @returns a `KubernetesManifest` construct representing the chart.
    */
-  addCdk8sChart(id: string, chart: Construct): KubernetesManifest;
+  addCdk8sChart(id: string, chart: Construct, options?: KubernetesManifestOptions): KubernetesManifest;
```

Comment:
The options argument should have always been here, otherwise we cannot customize the manifest with the options `KubernetesManifest` provides. 

Specifically, for this PR, these options are necessary for the integration test.